addons/account/wizard/account_move_reversal.py

```diff
@@ -96,6 +96,7 @@ def _compute_from_moves(self):
 
     def _prepare_default_reversal(self, move):
         reverse_date = self.date if self.date_mode == 'custom' else move.date
+        mixed_payment_term = move.invoice_payment_term_id.id if move.invoice_payment_term_id and move.company_id.early_pay_discount_computation == 'mixed' else None
```

Comment:
Why don't we always set it then?
Doesn't really make sense to only set it in one particular case.

Doesn't really make sense to have it set on the credit not either.

Reply:
Indeed it doesn't really make sense to have a payment term on a credit note, however...
Since the tax is computed on the discounted amount, the invoice totals are strange eg 100.00 + 15% = 112.70 
So if the payment term is visible (in the exceptional case of a mixed payment term) it helps to make sense of it. Discussed with @chklop